src/hoa-renderer.js

```diff
@@ -47,12 +47,12 @@ function HOARenderer(context, options) {
   this._renderingMode = 'ambisonic';
   this._ambisonicOrder = 3;
 
-  if (options) {
-    if (options.HRIRUrl)
+  if (options !== undefined) {
```

Comment:
Yes, my question was how `if (options !== undefined)` is different from `if (options)`. I think they are compatible. If you want a more rigorous check, we should just do type checking and property sniffing. (and I prefer not to.)